lib/private/twilio/send-message.js

```diff
@@ -0,0 +1,105 @@
+module.exports = {
+
+
+  friendlyName: 'Send message',
+
+
+  description: 'Send a message using the Twilio SMS API',
+
+
+  moreInfoUrl: 'https://www.twilio.com/docs/sms',
+
+
+  inputs: {
+
+    accountSid: {
+      example: 'DZe5eafd3c69b1e74example5852c04a9102',
+      description: 'The "account sid" associated with your Twilio account.',
+      whereToGet: {
+        url: 'https://www.twilio.com/console',
+      },
+      required: true
+    },
+
+    authToken: {
+      example: 'Dafd4example5852c3c69bZe5e1e704a9102',
+      description: 'The "auth token" associated with your Twilio account.',
+      whereToGet: {
+        url: 'https://www.twilio.com/console',
+      },
+      required: true
+    },
+
+    from: {
+      example: '+16155551234',
+      description: 'The \'From\' phone number you\'d like to use to send the SMS.',
+      extendedDescription: 'This phone number is assigned to you by Twilio. If omitted, the first available number will be used.',
+      required: true
+    },
+
+    to: {
+      example: '+16155556789',
+      description: 'The \'To\' phone number the message is sending to.',
+      required: true
+    },
+
+    message: {
+      example: 'Hey there friend!',
+      description: 'The body of the message being sent',
+      extendedDescription: 'Optional if a `mediaUrl` is specified.'
+    },
+
+    mediaUrl: {
+      example: 'https://sailsjs.com/logo.png',
+      description: 'The URL of a gif, png, or jpeg image.',
+      extendedDescription: 'An alternative to `message`.'
```

Comment:
wait, can I use both or not?

Reply:
Also, would you mind adding some more info on what this means and how it's delivered?  e.g. does Twillio say it is an MMS?  (maybe a moreInfoUrl pointing at Twillio's explanation)

Reply:
Took a stab at this, would love your thoughts when you re-review. (Won't mark as resolved yet.)